.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -47,23 +47,7 @@ body:
 - type: dropdown
   attributes:
     label: Issue Type
-    description: >
-      Please select the single available option in the drop-down.
-
-      <details>
-        <summary>
-          <em>Why?</em>
-        </summary>
-
-        We would do it by ourselves but unfortunately, the current
-        edition of GitHub Issue Forms Alpha does not support this yet 🤷
-
-
-        _We will make it easier in the future, once GitHub
-        supports dropdown defaults. Promise!_
-
-      </details>
-    # FIXME: Once GitHub allows defining the default choice, update this
+    description: This is a marker for out automatic bot. Do not change it.
```

Comment:
```suggestion
    description: This is a marker for our automatic bot. Do not change it.
```

Reply:
Thanks!

Reply:
Though, it would need to be fixed in all 3 files..